qt/python/mantidqt/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -315,7 +314,17 @@ def open_file_in_new_tab(self, filepath, startup=False):
         with open(filepath, "r") as code_file:
             content = code_file.read()
 
-        self.append_new_editor(content=content, filename=filepath)
+        matching_index = -1
+        for i in range(self.editor_count):
+            if self.editor_at(i).filename == filepath:
+                matching_index = i
+                break
+
+        if matching_index > -1:
+            self._tabs.setCurrentWidget(self.editor_at(matching_index))
```

Comment:
```suggestion
                matching_index = i
                break
                
        # If the file is currently open, select that tab
        if matching_index > -1:
            self._tabs.setCurrentWidget(self.editor_at(matching_index))
```
Thought a comment might be nice here